cfgov/v1/atomic_elements/organisms.py

```diff
@@ -875,6 +875,14 @@ class FilterControls(BaseExpandable):
     ], default='filterable-list')
     title = blocks.BooleanBlock(default=True, required=False,
                                 label='Filter Title')
+    no_posts_message = blocks.CharBlock(
+        required=False,
+        help_text='When there are no posts of this type, display '
+                  'this message instead of filter controls. (header)')
+    no_posts_explanation = blocks.CharBlock(
+        required=False,
+        help_text='When there are no posts of this type, display '
+                  'this message instead of filter controls. (paragraph)')
```

Comment:
I think it might help editors to know to expect a cf-notifications-style message if we reword these `help_text`s slightly and add a link. How about something like:

```py
from django.utils.safestring import mark_safe

… 

    no_posts_message = blocks.CharBlock(
        required=False,
        help_text=mark_safe(
            'Message for the <a href="https://cfpb.github.io/'
            'capital-framework/components/cf-notifications/'
            '#recommended-notification-patterns">notification</a> '
            'that will be displayed instead of filter controls '
            'if there are no posts to filter.'
        )
    )
    no_posts_explanation = blocks.CharBlock(
        required=False,
        help_text='Additional explanation for the notification that '
                  'will be displayed if there are no posts to filter.'
    )
```

Reply:
Good suggestion. I've made that change and updated the Wagtail screenshot above.